tracer/src/Datadog.Trace/DataStreamsMonitoring/DataStreamsWriter.cs

```diff
@@ -48,6 +55,30 @@ internal class DataStreamsWriter
             period: bucketDurationMs);
     }
 
+    /// <summary>
+    /// Public for testing only
+    /// </summary>
+    public event EventHandler<EventArgs>? FlushComplete;
+
+    private enum SupportState
```

Comment:
nit: Probably unnecessary, but could you assign integers to each value?

Reply:
Looks like Steven asked the same thing and you addressed it so I'll resolve. I'm not sure why I didn't see that comment show up when I went to comment though, strange 🤔